resources/conf/config.toml.template

```diff
@@ -345,12 +345,20 @@ enabled = true
   gatewayGeneratorImpl = "org.wso2.carbon.apimgt.common.gateway.jwtgenerator.APIMgtGatewayJWTGeneratorImpl"
   # Custom Claim Retriever to add custom claims into JWT
   claimsExtractorImpl = "org.wso2.carbon.apimgt.impl.token.ExtendedDefaultClaimsRetriever"
-  # Public certificate for generated JWT
-  publicCertificatePath = "/home/wso2/security/truststore/mg.pem"
-  # Private key path used for JWT generation
-  privateKeyPath = "/home/wso2/security/keystore/mg.key"
   # Token expiry time in seconds
   tokenTTL = 3600
+  # Whether to use kid property or x5t to identify backend JWTs
+  useKidProperty = false
```

Comment:
If we enable this bydefault, I think we can omit the property from here.

Reply:
ack